### DISCUSSION TITLE
Exit code -1073741511 (0x3FFFFEC7) since 2.38 on docker

### DISCUSSION THREAD

**Discussion by viceice** [git-for-windows/git#4091] [Q&A] [answered]
Any idea why i get `-1073741511` exit code when upgrading mingit on my docker images[^1]?

I'm using it on `mcr.microsoft.com/powershell:7.2-nanoserver-ltsc2022` and `mcr.microsoft.com/powershell:7.2-nanoserver-1809`.
Version 2.37.3 works fine.

I also see no output at all :-/

[^1]: https://github.com/visualon/docker-drone-git

**Comment by rimrul** [comment, marked as answer]
> `mcr.microsoft.com/powershell:7.2-nanoserver-ltsc2022` and `mcr.microsoft.com/powershell:7.2-nanoserver-1809`.

Sounds a lot like #4052

**Reply by viceice** [reply]
yes. 😕

**Reply by dscho** [reply]
FWIW the latest [snapshot](https://wingit.blob.core.windows.net/files/index.html) has a fix for this.
